package/carbonio-preview-start.sh

```diff
@@ -4,18 +4,6 @@
 # 
 # SPDX-License-Identifier: AGPL-3.0-only
 
-# This file sole purpose is to change working directory and execute gunicorn.
```

Comment:
Why this comment is no longer needed?

Reply:
Ok, so to recap: we could execute the service in a different way (without this script) but this will be for a future task and maintaining this comment could lead to misunderstanding since it refers to outdated facts. If it is right we can resolve this conversation :+1: !